pkg/testutil/compose.go

```diff
@@ -50,15 +50,15 @@ func (cd *ComposeDir) CleanUp() {
 	os.RemoveAll(cd.dir)
 }
 
-func NewComposeDir(t testing.TB, dockerComposeYAML string) *ComposeDir {
+func NewComposeDir(t testing.TB, dockerComposeYAML string, yamlName string) *ComposeDir {
```

Comment:
maybe this variable can be namedm as yamlBasePath

Reply:
LGTM